### ISSUE TITLE
English version

### ISSUE THREAD

**Issue by prein** [metalance/stundensatzrechner#2]
Great tool! Do you think it would be possible to have it work in other languages?

**Comment by metalance** [comment]
Of course, any time. What do you have in mind?
The tool uses a subset of UI elements of the open source library OpenUI5. So it's actually just hooking up to the events of the sliders in order to recalculate the quote. In case you want to migrate it to another language/platform, it should offer similar UI elements. Otherwise you would have to develop the UI stuff on your own. That would be a lot more work compared to just implementing the calculation logic.

**Comment by metalance** [comment]
oh, sorry. I missunderstood. I was thinking of other programming languages. I guess you were talking about displayig the labels and texts in English instead of German only. Yes, this would be possible.

**Comment by prein** [comment]
Yes, correct, I thought of slider labels. So that for example "Bruttogehalt bei vergleichbarer Festanstellung" becomes "Gross salary for a comparable permanent position".
